security/s2a/internal/crypter/counter.go

```diff
@@ -0,0 +1,37 @@
+package crypter
+
+import (
+	"errors"
+)
+
+// counterLen is the byte length of the counter.
+const counterLen = 8
+
+// counter is a 64-bit, little-endian counter.
```

Comment:
I think we can remove `little-endian` now that there is no more array.

Reply:
Done